### ISSUE TITLE
Need Info! Static server

### ISSUE THREAD

**Issue by michailVestnik** [l2ig/aqua#34]
Hi! I'm new to backend, I hope for your help!
I run this example, but CSS style dont work!
Of course styles are connected in HTML

```

import Aqua from 'https://deno.land/x/aqua/mod.ts'

const app = new Aqua(3100)

app.get('/', async (req) => {
    return await app.render('static/index.html')
})

```

**Comment by l2ig** [comment]
You will also need to serve the other resources (like CSS files, etc.).
```typescript
app.serve("static", "/");
```
should do the trick.

If this doesn't fix your problem, please provide your full code.

**Comment by michailVestnik** [comment]
Thank you!
my problem is solved.
P.S Please add wasm in MIME-types / application/wasm

**Comment by l2ig** [comment]
Published in v1.0.17. :)
